src/ui/popupmenu/completion_item_widget.rs

```diff
@@ -72,12 +85,14 @@ impl CompletionItemWidgetWrap {
         let row = gtk::ListBoxRow::new();
         row.add(&grid);
 
-        add_css_provider!(css_provider, grid, kind, word, info, row, menu);
+        add_css_provider!(css_provider, grid, word, image, info, row, menu);
 
+        let kind = item.kind.clone();
```

Comment:
Instead of doing a implicit clone, require a owner ship of the value and do the clone on a higher level.

Reply:
@vhakulinen Do you mean using `to_owned()` instead, or doing something else? (I am not entirely familiar with the way cloning works, so that's why I'm asking.)

Reply:
Oh, sorry. This one was a brainfart on my part. Ignore this comment.